src/rest/StreamEndpoints.js

```diff
@@ -60,52 +45,70 @@ export async function createStream(props, apiKey = this.options.apiKey) {
         apiKey,
         {
             method: 'POST',
-            body: JSON.stringify(props),
+            data: props,
         },
     )
     return json ? new Stream(this, json) : undefined
 }
 
 export async function getOrCreateStream(props, apiKey = this.options.apiKey) {
-    let json
-
-    // Try looking up the stream by id or name, whichever is defined
-    if (props.id) {
-        json = await this.getStream(props.id, apiKey)
-    } else if (props.name) {
-        json = await this.getStreamByName(props.name, apiKey)
-    }
+    let stream = await (props.id ? this.getStream(props.id, apiKey) : this.getStreamByName(props.name, apiKey))
 
     // If not found, try creating the stream
-    if (!json) {
-        json = await this.createStream(props, apiKey)
-        debug('Created stream: %s (%s)', props.name, json.id)
+    if (!stream) {
+        stream = await this.createStream(props, apiKey)
     }
 
-    // If still nothing, throw
-    if (!json) {
-        throw new Error(`Unable to find or create stream: ${props.name}`)
-    } else {
-        return new Stream(this, json)
-    }
+    debug('Created stream: %s (%s)', props.name, stream.id)
```

Comment:
This message should only be shown if a stream was really created (not fetched), just like before.